### PR TITLE
feat: add CSS SVG d and fill-* properties

### LINKED ISSUE #210: Project: Add missing "widely available" CSS attributes
### Problem statement

There are 29 "widely supported" baseline CSS properties missing from MDN documentation. These CSS properties need to be documented.

### Proposed solutions

add missing pages.

### Task list

- [x] [`clip-rule`](https://developer.mozilla.org/en-US/docs/Web/CSS/clip-rule)
- [x] [`color-interpolation-filters`](https://developer.mozilla.org/en-US/docs/Web/CSS/color-interpolation-filters)
- [x] [`cx`](https://developer.mozilla.org/en-US/docs/Web/CSS/cx)
- [x] [`cy`](https://developer.mozilla.org/en-US/docs/Web/CSS/cy)
- [x] [`dominant-baseline`](https://developer.mozilla.org/en-US/docs/Web/CSS/dominant-baseline)
- [x] [`fill`](https://developer.mozilla.org/en-US/docs/Web/CSS/fill)
- [x] [`fill-opacity`](https://developer.mozilla.org/en-US/docs/Web/CSS/fill-opacity)
- [x] [`fill-rule`](https://developer.mozilla.org/en-US/docs/Web/CSS/fill-rule)
- [x] [flood-color](https://drafts.fxtf.org/filter-effects-1/#FloodColorProperty)
- [x] [flood-opacity](https://drafts.fxtf.org/filter-effects-1/#FloodOpacityProperty)
- [x] [lighting-color](https://drafts.fxtf.org/filter-effects-1/#LightingColorProperty)
- [x] [`marker`](https://developer.mozilla.org/en-US/docs/Web/CSS/marker) 
- [x] [`marker-end`](https://developer.mozilla.org/en-US/docs/Web/CSS/marker-end)
- [x] [`marker-mid`](https://developer.mozilla.org/en-US/docs/Web/CSS/marker-mid)
- [x] [`marker-start`](https://developer.mozilla.org/en-US/docs/Web/CSS/marker-start)
- [x] [`r`](https://developer.mozilla.org/en-US/docs/Web/CSS/r)
- [x] [`rx`](https://developer.mozilla.org/en-US/docs/Web/CSS/rx)
- [x] [`ry`](https://developer.mozilla.org/en-US/docs/Web/CSS/ry)
- [x] [`shape-rendering`](https://developer.mozilla.org/en-US/docs/Web/CSS/shape-rendering)
- [x] [`stop-color`](https://developer.mozilla.org/en-US/docs/Web/CSS/stop-color)
- [x] [`stop-opacity`](https://developer.mozilla.org/en-US/docs/Web/CSS/stop-opacity)
- [x] [`stroke`](https://developer.mozilla.org/en-US/docs/Web/CSS/stroke)
- [x] [`stroke-dasharray`](https://developer.mozilla.org/en-US/docs/Web/CSS/stroke-dasharray)
- [x] [`stroke-dashoffset`](https://developer.mozilla.org/en-US/docs/Web/CSS/stroke-dashoffset)
- [x] [`stroke-linecap`](https://developer.mozilla.org/en-US/docs/Web/CSS/stroke-linecap)
- [x] [`stroke-linejoin`](https://developer.mozilla.org/en-US/docs/Web/CSS/stroke-linejoin)
- [x] [`stroke-miterlimit`](https://developer.mozilla.org/en-US/docs/Web/CSS/stroke-miterlimit)
- [x] [`stroke-opacity`](https://developer.mozilla.org/en-US/docs/Web/CSS/stroke-opacity)
- [x] [`stroke-width`](https://developer.mozilla.org/en-US/docs/Web/CSS/stroke-width)
- [x] [`text-anchor`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-anchor)
- [x] [`vector-effect`](https://developer.mozilla.org/en-US/docs/Web/CSS/vector-effect)
- [x] [`word-wrap`](https://developer.mozilla.org/en-US/docs/Web/CSS/word-wrap)
- [x] [`x`](https://developer.mozilla.org/en-US/docs/Web/CSS/x)
- [x] [`y`](https://developer.mozilla.org/en-US/docs/Web/CSS/y)

### What did you expect to see?

these properties documented

### Do you have any supporting links, references, or citations?

https://openwebdocs.github.io/web-docs-backlog/baseline/

### Priority assessment

_No response_

### More information

_No response_

### PR DESCRIPTION
**Pull request by estelle**
### Description

add `d`, `fill`, `fill-opacity`, and `fill-rule` properties

### Motivation

these reference pages are being added to MDN docs under CSS

### Additional details

https://svgwg.org/svg2-draft/painting.html#FillProperties
https://svgwg.org/svg2-draft/paths.html#TheDProperty
also made alphabetical

### Related issues and pull requests

Add missing CSS properties: https://github.com/mdn/content/issues/34763
Fixes: https://github.com/openwebdocs/project/issues/210
addresses: https://github.com/mdn/data/issues/750

### CODE REVIEW

File: css/properties.json [hunk 1]
```diff
@@ -4733,6 +4749,54 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/field-sizing"
   },
+  "fill": {
+    "syntax": "none | <color> | <url> [none | <color>]? | context-fill | context-stroke",
+    "media": "visual",
+    "inherited": true,
+    "animationType": "byComputedValueType",
```
Comment (bsmth) [inline css/properties.json:4756]: Should we be listing:

>the fill painting properties—[fill-color](https://drafts.fxtf.org/fill-stroke-3/#propdef-fill-color), [fill-image](https://drafts.fxtf.org/fill-stroke-3/#propdef-fill-image), [fill-origin](https://drafts.fxtf.org/fill-stroke-3/#propdef-fill-origin), [fill-position](https://drafts.fxtf.org/fill-stroke-3/#propdef-fill-position), [fill-size](https://drafts.fxtf.org/fill-stroke-3/#propdef-fill-size), and [fill-repeat](https://drafts.fxtf.org/fill-stroke-3/#propdef-fill-repeat)

Similar to https://github.com/mdn/data/pull/657/files?

File: css/properties.json [hunk 2]
```diff
@@ -4733,6 +4749,54 @@
...
+    "appliesto": "limitedSVGElementsShapeText",
+    "computed": "asColorOrAbsoluteURL",
+    "order": "perGrammar",
+    "status": "standard",
+    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/fill"
+  },
+  "fill-opacity": {
+    "syntax": "<alpha-value>",
+    "media": "visual",
+    "inherited": false,
```
Comment (bsmth) [inline css/properties.json:4771]: I think this inherits: https://drafts.fxtf.org/fill-stroke-3/#fill-opacity or am I looking at the wrong place?

File: css/properties.json (hunk 1 above)
Comment (estelle) [reply css/properties.json:4756]: no. 
`<paint> is defined as color and fallback, not background image.
[https://svgwg.org/svg2-draft/painting.html#FillProperties is ](https://svgwg.org/svg2-draft/painting.html#SpecifyingPaint)

origin, position, size, and repeat are not part of paint in any implementation

File: css/properties.json (hunk 2 above)
Comment (estelle) [inline css/properties.json:4771]:
```suggestion
    "inherited": true,
```

Comment (estelle) [reply css/properties.json]: good catch.
but https://svgwg.org/svg2-draft/painting.html#StrokeOpacity is the better spec.

Comment (bsmth) [reply css/properties.json]: Right, gotcha, this is confusing with the SVG/CSS crossover

**Review by bsmth — APPROVED** [review]
Thanks, Estelle, I think we're good to go!

**Pull request merged**